pydatalab/pydatalab/models/people.py

```diff
@@ -91,6 +92,21 @@ def add_missing_type(cls, v):
     def set_default_type(cls, _):
         return "people"
 
+    @validator("display_name")
+    def validate_display_name_length(cls, v):
+        """Validate that the display name."""
+        if len(v) > 150:
+            raise ValueError("Display name must be at most 150 characters long.")
```

Comment:
```suggestion
        max_len = 150
        if len(v) > max_len:
            raise ValueError(f"Display name must be at most {max_len} characters long.")
```

Minor tweak to make it easier to update